### PR TITLE
Gt 428 Update Adobe Analytics

### PR DESCRIPTION
**Pull request by ryancarlson**
- import Adobe Analytics as a cocoapod instead of a library
- use different config files for debug and release builds
- change Adobe app ID in both config files
- remove `GodTools :` prefix in screen name reporting
- minor compatibility changes required to build in XCode9
   - RealmSwift and XML parsing library

### CODE REVIEW

File: godtools/Views/TractElements/TractTabs+ChildSetup.swift [hunk 1]
```diff
@@ -19,7 +19,7 @@ extension TractTabs {
             for node in item.children {
                 if self.xmlManager.parser.nodeIsLabel(node: node) {
                     if let textNode = self.xmlManager.parser.getTextContentFromElement(node) {
-                        self.tabsProperties().options.append(textNode.text!)
+                        self.tabsProperties().options.append(textNode.text)
```
Comment (pablomarti) [inline godtools/Views/TractElements/TractTabs+ChildSetup.swift]: Why was the `!` removed?

File: godtools/XMLManagement/XMLManager.swift [hunk 2]
```diff
@@ -26,7 +26,7 @@ class XMLManager: NSObject {
             properties[attribute.name] = attribute.text
         }
         
-        if data.element?.text != nil && data.element?.text?.trimmingCharacters(in: .whitespaces) != "" {
+        if data.element?.text != nil && data.element?.text.trimmingCharacters(in: .whitespaces) != "" {
```
Comment (pablomarti) [inline godtools/XMLManagement/XMLManager.swift]: Why was the `?` removed?

File: godtools/Views/TractElements/TractTabs+ChildSetup.swift (hunk 1 above)
Comment (ryancarlson) [reply godtools/Views/TractElements/TractTabs+ChildSetup.swift]: This was a required as part of building in XCode 9. I'm not sure why, but the build would not pass w/o this change.

File: godtools/XMLManagement/XMLManager.swift (hunk 2 above)
Comment (ryancarlson) [reply godtools/XMLManagement/XMLManager.swift]: Same as the other comment. This was a required as part of building in XCode 9. I'm not sure why, but the build would not pass w/o this change.

**Comment by ryancarlson** [comment]
Thanks! Yeah those two changes showed up as build errors in XCode 9, but worked in XCode 8. I had to make those changes to get things working in the new environment, but I did not dig in deeply to see why.

File: godtools/XMLManagement/XMLManager.swift (hunk 2 above)
Comment (pablomarti) [reply godtools/XMLManagement/XMLManager.swift]: Ok, thanks!

**Pull request merged**